.github/workflows/build.yaml

```diff
@@ -496,15 +496,15 @@ jobs:
       - name: Install build dependencies
         run: |
           dnf install -y 'dnf-command(builddep)'
-          dnf builddep -y qpid-dispatch-router --setopt=install_weak_deps=False
+          dnf builddep -y skupper-router --setopt=install_weak_deps=False
```

Comment:
This won't work; the Fedora package is still named `qpd-dispatch-router`. Using Fedora package for the purpose seemed expedient at the time. We should IMO merge https://github.com/skupperproject/skupper-router/pull/115 which removes dependence on the Fedora package.

Reply:
@jiridanek ah - thanks, I was a little lost there.

I've approved #115 and I'll mark this PR as a draft until I can rebase on that change, thanks.